php/composer.json

```diff
@@ -0,0 +1,13 @@
+{
+  "require":{
+    "google/protobuf": "^3.3",
+    "grpc/grpc": "^1.42",
+    "hanson/foundation-sdk": "^4.0"

```

Comment:
It seems that foundation-sdk is not compatiable with PHP 8+, upgrade it to 5.x?

Reply:
This component is used to assist in the development of the SDK, it is not used yet, if you do not need it later, you can directly remove it from the composer.json